Source/RapyutaSimulationPlugins/Public/Robots/Turtlebot3/TurtlebotBurger.h

```diff
@@ -11,6 +11,7 @@
 
 DECLARE_LOG_CATEGORY_EXTERN(LogTurtlebotBurger, Log, All);
 
+class URR2DLidarComponent;

```

Comment:
please avoid forward declarelation
https://google.github.io/styleguide/cppguide.html#Forward_Declarations

Reply:
Right, thanks, kinda my habits to avoid cyclic depedency, which is not really required in this case.